packages/jellyfish-transaction/src/script/dftx/dftx_balance.ts

```diff
@@ -3,17 +3,22 @@ import { BufferComposer, ComposableBuffer } from '@defichain/jellyfish-buffer'
 import { Script } from '../../tx'
 import { CScript } from '../../tx_composer'
 
-export interface TokenBalance {
+/**
+ * Known as "struct CBalances" in cpp.
+ */
+export interface TokenBalanceUInt32 {
   token: number // ---------------------| 4 bytes unsigned
```

Comment:
```suggestion
  token: number // ---------------------| VarInt{MSB-b128}
```
Theoretically MAX SAFE INTEGER in base128

Reply:
Nope, there is another specification for "TokenBalanceUInt32" hence the rename. It's UInt32, hence 4 bytes unsigned.